build.sbt

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-import Dependencies._
-import SparkVersionAxis._
-import JacocoSetup._
+import Dependencies.*
+import SparkVersionAxis.*
+import JacocoSetup.*
```

Comment:
Actually I didn't change this. I have no idea how that happened. And it probably doesn't matter since it still imports everything. I know that `*` symbol is used in Scala 2.13 and later in many places where previously `_` was used.

Reply:
Since we still have to generally support Scala 2.12 I would rather move it back to `_` - https://github.com/scala/scala/releases/tag/v2.13.7 it looks like it's more like a Scala 3 thing, with support in this 2.13 version

Reply:
ok, changed